app/graphql/types/registration_type.rb

```diff
@@ -19,6 +19,7 @@ def room
     end
     field :start_time, GraphQL::Types::ISO8601DateTime, null: true
     field :end_time, GraphQL::Types::ISO8601DateTime, null: true
+    field :effective_end_time, GraphQL::Types::ISO8601DateTime, null: true
```

Comment:
Just making a performance note: this will likely be an N+1 query, since it loads associated tables in order to compute this quantity...